src/sidebar/components/focused-mode-header.js

```diff
@@ -4,42 +4,59 @@ const { createElement } = require('preact');
 
 const useStore = require('../store/use-store');
 
+const SvgIcon = require('./svg-icon');
+
 /**
  * Render a control to interact with any focused "mode" in the sidebar.
  * Currently only a user-focus mode is supported but this could be broadened
  * and abstracted if needed. Allow user to toggle in and out of the focus "mode."
  */
 function FocusedModeHeader() {
-  const store = useStore(store => ({
-    actions: {
-      setFocusModeFocused: store.setFocusModeFocused,
-    },
-    selectors: {
-      focusModeFocused: store.focusModeFocused,
-      focusModeUserPrettyName: store.focusModeUserPrettyName,
-    },
+  const actions = useStore(store => ({
+    setFocusModeFocused: store.setFocusModeFocused,
+  }));
+  const selectors = useStore(store => ({
+    focusModeFocused: store.focusModeFocused(),
+    focusModeHasUser: store.focusModeHasUser,
```

Comment:
You need to extract the actual data out of the store here (`store.focusModeHasUser()`) rather than just a reference to the method (`store.focusModeHasUser`), otherwise `useStore` can't tell if the data you are interested in changed or not.